package.json

```diff
@@ -21,9 +21,19 @@
   "main": "./dist/js/okta-sign-in.entry.js",
   "scripts": {
     "clean": "rimraf target && rimraf dist && rimraf build2",
-    "lint:report": "grunt lint && stylelint assets/sass/ > build2/OSW-stylelint-checkstyle.xml --custom-formatter node_modules/stylelint-checkstyle-formatter || true && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
-    "lint": "grunt lint && stylelint assets/sass/ && eslint .",
-    "test:dev": "grunt generate-config && karma start",
+    "generate-phone-codes": "node buildtools generate-phone-codes",
+    "generate-config": "node buildtools generate-language-config",
+    "generate-jsonp": "node buildtools generate-language-jsonp",
+    "retirejs": "retire --jspath src -p",
```

Comment:
will this work in all environments? I don't see `retire` in the list of `devDependencies` or `dependencies`... we still have `grunt-retire` there though... maybe we need to remove `grunt-retire` and add `retire`?

Reply:
good catch!!
I probably mess up when merging where there shall be changes that remove `grunt-retire` and add `retire` as devDependency.

It works now because `grunt-retire` pull `retire` as dependency.